config/psyclone.cfg

```diff
@@ -59,8 +59,7 @@ access_mapping = gh_read:read, gh_write: write, gh_rw: readwrite,
 # remove the need for a halo exchange call.
 COMPUTE_ANNEXED_DOFS = false
 
-access_mapping = gh_read:read, gh_write: write, gh_readwrite: readwrite,
-                 gh_inc: inc, gh_sum: sum
+access_mapping = gh_read:read, gh_write: write, gh_readwrite: readwrite, gh_inc: inc, gh_sum: sum
```

Comment:
Is this change needed? If not, please undo!

Reply:
Oops. Reverted.